### DISCUSSION TITLE
When opening a speed dial, how to open it in current tab?

### DISCUSSION THREAD

**Discussion by m3r1k** [fastaddons/GroupSpeedDial#55] [Q&A]
When opening a speed dial, how do I open the page in the same tab as the groupspeeddial?

**Comment by fastaddons** [comment]
Hello,
See the Options page / Dials tab / "Always open in New tab" - at the bottom, make sure it's not enabled.
And if it's not enabled, then your browser is set to open links in new tabs. In that case, you need to change it in your browser.

**Comment by BobHD** [comment]
I like opening in a new tab by default, but it would be nice if the first item on the right-click menu were "Open in TAB".
This option exists on the now defunct FVD Speedial, so it's possible without changing the Firefox settings.
